pkg/storage/BUILD.bazel

```diff
@@ -140,6 +140,7 @@ go_test(
     args = ["-test.timeout=295s"],
     data = glob(["testdata/**"]),
     embed = [":storage"],
+    exec_properties = {"Pool": "medium"},
     shard_count = 2,
     tags = ["no-remote-exec"],
```

Comment:
are you aware of this?

Reply:
`tags = ["no-remote-exec"],`

Reply:
Yes. Following up from #109176, it seems this test needs more resources. I'll follow up with removing the `no-remote-exec` later.